website/docs/sharedlibtype.md

```diff
@@ -10,6 +10,7 @@ sharedlibtype "value"
 * `OSXBundle` - needs documentation.
 * `OSXFramework` - needs documentation.
 * `XCTest` - needs documentation.
+* `XCUITest`- needs documentation.
```

Comment:
If you're able to add documentation to describe this, do you mind doing so? That's basically a TODO for us when we updated our docs to get documentation added to fields.

Reply:
You happy with me updating that whole page then? It all needs documentation?

Reply:
That would be great.

Reply:
Sure, I'll get to it after the holidays, wasn't sure if you're OK wiht doing this in same commit.

Reply:
Hmm, good point about same PR. It may make sense to just touch this line in this PR, then do the rest in a follow up.  I don't have a huge preference either way, since the PR will stay small.

Reply:
> Sure, I'll get to it after the holidays, wasn't sure if you're OK with doing this in same PR.

Not sure when the holidays end for you.  I'm trying to start prepping a new release build of Premake. Will this be ready by the weekend?

Reply:
No, I'm back on 5th, so that week would be earliest. No rush on my side, we run custom fork until it gets merged. Thanks.